rds/aurora/mysql/serverless-v1/README.md

```diff
@@ -0,0 +1,79 @@
+# Aurora MySQL Serverless V1
+
+This Acorn creates an Aurora MySQL serverless cluster running on AWS RDS service. This Acorn is best used when you have a variable workload that sometimes is almost completely unused. If you have a stable workload, you should evaluate the cost of serverless vs. cluster based Aurora MySQL.
```

Comment:
Isn't serverless V2 preferred over V1? Perhaps we should let the user know that here.